src/components/graphqlFormGenerator/FormInput.vue

```diff
@@ -141,6 +99,70 @@ export default {
 
       return ret
     }
+  },
+
+  render (createElement) {
```

Comment:
Out of interest what was the motivation for moving from `<template>` to `render()`? I can't see anything that looks like it couldn't have been done with `<template>`.

Reply:
The template-based way became a bit awkward for passing in slot props to the `append-outer` slot, due to an issue in Vuetify v2 (sorry can't find the GH link). Basically, doing it with a render function avoids the need to duplicate the slot with an if-else.